engine/src/main/java/org/terasology/rendering/opengl/GLSLShader.java

```diff
@@ -309,7 +312,7 @@ private void dumpCode(int type, Set<ShaderProgramFeature> features, String sourc
         }
 
         // Dump all final shader sources to the log directory
-        final String strippedTitle = getUrn().toString().replace(":", "-");
+        final String strippedTitle = COLON.matcher(getUrn().toString()).replaceAll(Matcher.quoteReplacement("-"));
```

Comment:
Hmmmm. I'm seeing a longer line of code for dubious performance gains here. Even if the way you just did it was considerably faster, I don't think it's worth the loss of readability.